samples/matlab_experimental/plug_flow_reactor.m

```diff
@@ -8,19 +8,19 @@
 %
 % The reactor assumes that the flow follows the Ideal Gas Law.
 %
-% The governing equations used in this code can be referenced at:
+% The governing equations used in this code (see :ref:`PFR_solver.m
+% <sphx_glr_examples_matlab_experimental_PFR_solver.m>`) can be referenced at:
```

Comment:
The generated links are somewhat convoluted.

Reply:
They're certainly _long_. I realized that there is an alternative, which is to link to the page rather than the heading, which can be done like ``` :doc:`PFR_solver.m <PFR_solver>` ``` where the bracketed text needs to specify a relative path, e.g. for some of the Python links, we'd have something like ``` :doc:`reactor1.py <../reactors/reactor1>` ```.